src/install/install.php

```diff
@@ -297,6 +296,26 @@ private function validateAdmin(): bool
         return true;
     }
 
+    /**
+     * Validate currency information meets all required parameters.
+     *
+     * @return boolean
+     */
+     private function validateCurrency(): bool
```

Comment:
IMO, this should happen in javascript, not in the installation code, as that would lead to the user not being able to fix it before installing FOSSBilling. Also, validateCurrency isn't called anywhere in the installation code.